docs/asciidoc/import/webapis.adoc

```diff
@@ -62,6 +62,52 @@ or
 * `s3://endpoint:port/bucket/key?accessKey=accessKey&secretKey=secretKey`
 
 
+== Using Google Cloud Storage
+
+In order to use Google Cloud Storage you need that the following jars are included into the plugin dir:
+
+* api-common-1.8.1.jar
```

Comment:
lots of jars, can we look into providing a module that generates an uber jar for these? that we can make available?

Reply:
I created this issue #1297